.github/workflows/build_linux.yml

```diff
@@ -101,7 +101,7 @@ jobs:
             cmd_deps: sudo apt-get install -y -qq valgrind libmbedtls-dev mosquitto
             cmd_action: examples_valgrind MBEDTLS
           - build_name: "Valgrind Examples  with OpenSSL (gcc)"
-            cmd_deps: sudo apt-get install -y -qq valgrind openssl mosquitto
+            cmd_deps: sudo apt-get install -y -qq valgrind openssl libmbedtls-dev mosquitto
```

Comment:
Adding mbedTLS as a dependency  because mbedTLS is included when `UA_PUBSUB_ENCRYPTION` is enabled even if `UA_ENABLE_ENCRYPTION_OPENSSL` is set. Is there a reason behind this? 
 > eg. [CMakeLists.txt line 1088](https://github.com/open62541/open62541/blob/master/CMakeLists.txt#L1088)

Reply:
We currently only implement the pubsub security policies for mbedtls.
They need to be ported.
Either do that or exclude the pubsub / openssl combination.

Reply:
Okay. For now we could just do it like this  and add libmbedtls-dev as a dep?

We need to have Pubsub and OpenSSL as Cmake arguments in order for the examples to be built and executed.